main/config.go

```diff
@@ -49,7 +49,7 @@ const (
 	defaultReqBufSize = 20 // expected max. waiting time for accepted requests at a throughput of 3rps => ~7sec
 )
 
-var ENV string
+var EnvIsProd bool
```

Comment:
I think this doesnt have to be Public, we could use `isProduction` for more readable checks below

Reply:
Looking at the checks below, most of them will be `if !isProduction`, because behaviour is mostly changed specific to the development environments only.. Would it be a good idea to use `isDevelopment` and negate the checks?